explainer.md

```diff
@@ -86,19 +86,19 @@ This section describes the sequences of events that get fired on the EditContext
 dictionary TextUpdateEventInit {
     unsigned long updateRangeStart;
     unsigned long updateRangeEnd;
-    DOMString updateText;
-    unsigned long newSelectionStart;
-    unsigned long newSelectionEnd;
+    DOMString text;
+    unsigned long selectionStart;
+    unsigned long selectionEnd;
 };
 
 [Exposed=Window]
 interface TextUpdateEvent : Event {
     constructor(optional TextUpdateEventInit options = {});
     readonly attribute unsigned long updateRangeStart;
     readonly attribute unsigned long updateRangeEnd;
-    readonly attribute DOMString updateText;
-    readonly attribute unsigned long newSelectionStart;
```

Comment:
we have other occurrences in the spec, e.g. in examples and 3.1.1. it'll be good if we could update all in same PR